Makefile

```diff
@@ -113,69 +127,70 @@ misspell:
 		-name '*' \
 		-exec misspell -w {} \;
 
+## fmt : TBD.
 .PHONY: fmt
 fmt: add-headers python-env
 	@$(foreach var,$(PROJECTS) dev-tools $(PROJECTS_XPACK_MAGE),$(MAKE) -C $(var) fmt || exit 1;)
 	@# Cleans also python files which are not part of the beats
 	@$(FIND) -name "*.py" -exec $(PYTHON_ENV)/bin/autopep8 --in-place --max-line-length 120 {} \;
 
+## lint : TBD.
```

Comment:
Does this need to be `TBD`?

Reply:
There was no description, so I put TBD to agree whether we wanted to add something or not. I created one task to confirm with everyone what's the strategy. No strong opinion about what's to do.

Reply:
Ah, gotcha. Makes sense. 👍